### PR TITLE
Export/import tasks

### LINKED ISSUE #1300: Implement import/export of task definitions from Tasks tab

### PR DESCRIPTION
**Pull request by oodamien**
Resolves #1300

### CODE REVIEW

File: ui/src/app/tasks/tasks-utils/tasks-utils.service.ts [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+          if (c.composed) {
+            p.push(`${c.name}-`);
+          }
+          return p;
+        }, []);
+
+        let tasks = json.tasks.filter(item => {
+          return !composedName.find(name => item.name.startsWith(name))
+        });
+        //   .map(
```
Comment (jvalkeal) [inline ui/src/app/tasks/tasks-utils/tasks-utils.service.ts]: Commented code to zap?

**Comment by jvalkeal** [comment]
Other than that lgtm.

**Comment by ilayaperumalg** [comment]
LGTM, merging.

**Comment by ilayaperumalg** [comment]
Rebased and merged as 91521c69

**Pull request closed without merging**